cli/template/addons/trpc/auth-server-utils.ts

```diff
@@ -9,19 +9,30 @@ const t = initTRPC.context<Context>().create({
   },
 });
 
-export const createRouter = t.router;
+export const router = t.router;
 
-export const baseProcedure = t.procedure;
+/**
+ * Unprotected procedure
+ **/
+export const publicProcedure = t.procedure;
 
-export const authedProcedure = t.procedure.use(({ ctx, next }) => {
+/**
+ * Reusable middleware to ensure
+ * users are logged in
+ */
+const isAuthed = t.middleware(({ ctx, next }) => {
   if (!ctx.session || !ctx.session.user) {
     throw new TRPCError({ code: "UNAUTHORIZED" });
   }
   return next({
     ctx: {
```

Comment:
why is ctx not being spread in here anymore?

Reply:
Its not needed in v10, its doing an override so only the stuff you want to change need to be explicit